cypress/e2e/ModelizerPage/AllView/settings/git.settings.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Test git configuration dialog
+
+  Background:
+    Given I clear cache
+    And I set viewport size to "1536" px for width and "960" px for height
+    And I visit the "/"
+
+    When I click on "[data-cy=\"new-project\"]"
+    Then I expect current url is "/modelizer/project-[a-f0-9]{8}/model"
+    And  I extract "project-[a-f0-9]{8}" from url in field "projectName" of context
+    And I visit the "/#/modelizer/{{ projectName }}/text"
+
+  Scenario: Set git configuration for a project
```

Comment:
`Set git configuration for a project` -> `Set correct git configuration in the project`

Reply:
done